src/insexpand.c

```diff
@@ -3764,8 +3770,9 @@ get_next_default_completion(ins_compl_next_state_T *st, pos_T *start_pos)
 		&& start_pos->col  == st->cur_match_pos->col)
 	    continue;
 
-	ptr = ins_compl_get_next_word_or_line(st->ins_buf, st->cur_match_pos,
-							   &len, &cont_s_ipos);
+	if (len == 0)
```

Comment:
At first glance `len` may be uninitialized, let's use `in_fuzzy` .

Reply:
in process